### ISSUE TITLE
2025-01-22, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#59]
# Today's Arxiv Papers

(1) [InternVideo2.5: Empowering Video MLLMs with Long and Rich Context Modeling](http://arxiv.org/abs/2501.12386v1), 2025-01-21

(2) [Expertise elevates AI usage: experimental evidence comparing laypeople and professional artists](http://arxiv.org/abs/2501.12374v1), 2025-01-21

(3) [Is Long Context All You Need? Leveraging LLM's Extended Context for NL2SQL](http://arxiv.org/abs/2501.12372v1), 2025-01-21

(4) [Treefix: Enabling Execution with a Tree of Prefixes](http://arxiv.org/abs/2501.12339v1), 2025-01-21

(5) [Automatic Labelling with Open-source LLMs using Dynamic Label Schema Integration](http://arxiv.org/abs/2501.12332v1), 2025-01-21

(6) [VARGPT: Unified Understanding and Generation in a Visual Autoregressive Multimodal Large Language Model](http://arxiv.org/abs/2501.12327v1), 2025-01-21

(7) [LLM-Assisted Knowledge Graph Completion for Curriculum and Domain Modelling in Personalized Higher Education Recommendations](http://arxiv.org/abs/2501.12300v1), 2025-01-21

(8) [MoGERNN: An Inductive Traffic Predictor for Unobserved Locations in Dynamic Sensing Networks](http://arxiv.org/abs/2501.12281v1), 2025-01-21

(9) [Condor: Enhance LLM Alignment with Knowledge-Driven Data Synthesis and Refinement](http://arxiv.org/abs/2501.12273v1), 2025-01-21

(10) [FOCUS: First Order Concentrated Updating Scheme](http://arxiv.org/abs/2501.12243v1), 2025-01-21

(11) [InsTALL: Context-aware Instructional Task Assistance with Multi-modal Large Language Models](http://arxiv.org/abs/2501.12231v1), 2025-01-21

(12) [CDW-CoT: Clustered Distance-Weighted Chain-of-Thoughts Reasoning](http://arxiv.org/abs/2501.12226v1), 2025-01-21

(13) [Leveraging Large Language Models for Realizing Truly Intelligent User Interfaces](http://dx.doi.org/10.1145/3613905.3650949), 2025-01-21

(14) [You Can't Eat Your Cake and Have It Too: The Performance Degradation of LLMs with Jailbreak Defense](http://arxiv.org/abs/2501.12210v1), 2025-01-21

(15) [BiMarker: Enhancing Text Watermark Detection for Large Language Models with Bipolar Watermarks](http://arxiv.org/abs/2501.12174v1), 2025-01-21

(16) [AdaServe: SLO-Customized LLM Serving with Fine-Grained Speculative Decoding](http://arxiv.org/abs/2501.12162v1), 2025-01-21

(17) [Contextualizing Recommendation Explanations with LLMs: A User Study](http://arxiv.org/abs/2501.12152v1), 2025-01-21

(18) [Improving Influence-based Instruction Tuning Data Selection for Balanced Learning of Diverse Capabilities](http://arxiv.org/abs/2501.12147v1), 2025-01-21

(19) [Do LLMs Provide Links to Code Similar to what they Generate? A Study with Gemini and Bing CoPilot](http://arxiv.org/abs/2501.12134v1), 2025-01-21

(20) [Evaluating Efficiency and Engagement in Scripted and LLM-Enhanced Human-Robot Interactions](http://arxiv.org/abs/2501.12128v1), 2025-01-21

(21) [Can open source large language models be used for tumor documentation in Germany? -- An evaluation on urological doctors' notes](http://arxiv.org/abs/2501.12106v1), 2025-01-21

(22) [Dissecting the NVIDIA Hopper Architecture through Microbenchmarking and Multiple Level Analysis](http://arxiv.org/abs/2501.12084v1), 2025-01-21

(23) [Phishing Awareness via Game-Based Learning](http://arxiv.org/abs/2501.12077v1), 2025-01-21

(24) [EDoRA: Efficient Weight-Decomposed Low-Rank Adaptation via Singular Value Decomposition](http://arxiv.org/abs/2501.12067v1), 2025-01-21

(25) [PINNsAgent: Automated PDE Surrogation with Large Language Models](http://arxiv.org/abs/2501.12053v1), 2025-01-21

(26) [Harnessing Generative Pre-Trained Transformer for Datacenter Packet Trace Generation](http://arxiv.org/abs/2501.12033v1), 2025-01-21

(27) [Rate-Aware Learned Speech Compression](http://arxiv.org/abs/2501.11999v1), 2025-01-21

(28) [Linear Feedback Control Systems for Iterative Prompt Optimization in Large Language Models](http://arxiv.org/abs/2501.11979v1), 2025-01-21

(29) [Leveraging Graph Structures and Large Language Models for End-to-End Synthetic Task-Oriented Dialogues](http://arxiv.org/abs/2501.11977v1), 2025-01-21

(30) [Bridging Visualization and Optimization: Multimodal Large Language Models on Graph-Structured Combinatorial Optimization](http://arxiv.org/abs/2501.11968v1), 2025-01-21

(31) [A Hybrid Attention Framework for Fake News Detection with Large Language Models](http://arxiv.org/abs/2501.11967v1), 2025-01-21

(32) [TAD-Bench: A Comprehensive Benchmark for Embedding-Based Text Anomaly Detection](http://arxiv.org/abs/2501.11960v1), 2025-01-21

(33) [Proverbs Run in Pairs: Evaluating Proverb Translation Capability of Large Language Model](http://arxiv.org/abs/2501.11953v1), 2025-01-21

(34) [Webvs. LLMs: An Empirical Study of Learning Behaviors of CS2 Students](http://arxiv.org/abs/2501.11935v1), 2025-01-21

(35) [ALoFTRAG: Automatic Local Fine Tuning for Retrieval Augmented Generation](http://arxiv.org/abs/2501.11929v1), 2025-01-21

(36) [Integrate Temporal Graph Learning into LLM-based Temporal Knowledge Graph Model](http://arxiv.org/abs/2501.11911v1), 2025-01-21

(37) [Panoramic Interests: Stylistic-Content Aware Personalized Headline Generation](http://arxiv.org/abs/2501.11900v1), 2025-01-21

(38) [Med-R$^2$: Crafting Trustworthy LLM Physicians through Retrieval and Reasoning of Evidence-Based Medicine](http://arxiv.org/abs/2501.11885v1), 2025-01-21

(39) [From Drafts to Answers: Unlocking LLM Potential via Aggregation Fine-Tuning](http://arxiv.org/abs/2501.11877v1), 2025-01-21

(40) [Demons in the Detail: On Implementing Load Balancing Loss for Training Specialized Mixture-of-Expert Models](http://arxiv.org/abs/2501.11873v1), 2025-01-21

(41) [LLM-Agents Driven Automated Simulation Testing and Analysis of small Uncrewed Aerial Systems](http://arxiv.org/abs/2501.11864v1), 2025-01-21

(42) [EmbodiedEval: Evaluate Multimodal LLMs as Embodied Agents](http://arxiv.org/abs/2501.11858v1), 2025-01-21

(43) [Network-informed Prompt Engineering against Organized Astroturf Campaigns under Extreme Class Imbalance](http://arxiv.org/abs/2501.11849v1), 2025-01-21

(44) [A Survey on Memory-Efficient Large-Scale Model Training in AI for Science](http://arxiv.org/abs/2501.11847v1), 2025-01-21

(45) [Large Language Models with Human-In-The-Loop Validation for Systematic Review Data Extraction](http://arxiv.org/abs/2501.11840v1), 2025-01-21

(46) [Is your LLM trapped in a Mental Set? Investigative study on how mental sets affect the reasoning capabilities of LLMs](http://arxiv.org/abs/2501.11833v1), 2025-01-21

(47) [Benchmarking Large Language Models via Random Variables](http://arxiv.org/abs/2501.11790v1), 2025-01-20

(48) [Synthetic Data Can Mislead Evaluations: Membership Inference as Machine Text Detection](http://arxiv.org/abs/2501.11786v1), 2025-01-20

(49) [Glinthawk: A Two-Tiered Architecture for High-Throughput LLM Inference](http://arxiv.org/abs/2501.11779v1), 2025-01-20

(50) [The Value of Nothing: Multimodal Extraction of Human Values Expressed by TikTok Influencers](http://arxiv.org/abs/2501.11770v1), 2025-01-20

(51) [Poison-RAG: Adversarial Data Poisoning Attacks on Retrieval-Augmented Generation in Recommender Systems](http://arxiv.org/abs/2501.11759v1), 2025-01-20

(52) [Optimizing Pretraining Data Mixtures with LLM-Estimated Utility](http://arxiv.org/abs/2501.11747v1), 2025-01-20

(53) [Explain-Query-Test: Self-Evaluating LLMs Via Explanation and Comprehension Discrepancy](http://arxiv.org/abs/2501.11721v1), 2025-01-20

(54) [YouLeQD: Decoding the Cognitive Complexity of Questions and Engagement in Online Educational Videos from Learners' Perspectives](http://arxiv.org/abs/2501.11712v1), 2025-01-20

(55) [Towards Detecting Prompt Knowledge Gaps for Improved LLM-guided Issue Resolution](http://arxiv.org/abs/2501.11709v1), 2025-01-20

(56) [Trustformer: A Trusted Federated Transformer](http://arxiv.org/abs/2501.11706v1), 2025-01-20

(57) [Human services organizations and the responsible integration of AI: Considering ethics and contextualizing risk(s)](http://dx.doi.org/10.1080/15228835.2025.2457045), 2025-01-20

(58) [Advancing Language Model Reasoning through Reinforcement Learning and Inference Scaling](http://arxiv.org/abs/2501.11651v1), 2025-01-20

(59) [Early evidence of how LLMs outperform traditional systems on OCR/HTR tasks for historical records](http://arxiv.org/abs/2501.11623v1), 2025-01-20

(60) [Trojan Detection Through Pattern Recognition for Large Language Models](http://arxiv.org/abs/2501.11621v1), 2025-01-20

(61) [Conversation Routines: A Prompt Engineering Framework for Task-Oriented Dialog Systems](http://arxiv.org/abs/2501.11613v1), 2025-01-20

(62) [SR-FoT: A Syllogistic-Reasoning Framework of Thought for Large Language Models Tackling Knowledge-based Reasoning Tasks](http://arxiv.org/abs/2501.11599v1), 2025-01-20

(63) [Open Sourcing GPTs: Economics of Open Sourcing Advanced AI Models](http://arxiv.org/abs/2501.11581v1), 2025-01-20

(64) [Teaching Large Language Models to Regress Accurate Image Quality Scores using Score Distribution](http://arxiv.org/abs/2501.11561v1), 2025-01-20

(65) [PIKE-RAG: sPecIalized KnowledgE and Rationale Augmented Generation](http://arxiv.org/abs/2501.11551v1), 2025-01-20

(66) [Whose Boat Does it Float? Improving Personalization in Preference Tuning via Inferred User Personas](http://arxiv.org/abs/2501.11549v1), 2025-01-20

(67) [Generative AI and Large Language Models in Language Preservation: Opportunities and Challenges](http://arxiv.org/abs/2501.11496v1), 2025-01-20

(68) [Graph-defined Language Learning with LLMs](http://arxiv.org/abs/2501.11478v1), 2025-01-20

(69) [LLM supervised Pre-training for Multimodal Emotion Recognition in Conversations](http://arxiv.org/abs/2501.11468v1), 2025-01-20

(70) [Curiosity-Driven Reinforcement Learning from Human Feedback](http://arxiv.org/abs/2501.11463v1), 2025-01-20

(71) [Ontology Matching with Large Language Models and Prioritized Depth-First Search](http://arxiv.org/abs/2501.11441v1), 2025-01-20

(72) [RACCOON: A Retrieval-Augmented Generation Approach for Location Coordinate Capture from News Articles](http://arxiv.org/abs/2501.11440v1), 2025-01-20

(73) [One Does Not Simply Meme Alone: Evaluating Co-Creativity Between LLMs and Humans in the Generation of Humor](http://dx.doi.org/10.1145/3708359.3712094), 2025-01-20

(74) [A Survey on Diffusion Models for Anomaly Detection](http://arxiv.org/abs/2501.11430v1), 2025-01-20

(75) [Agent-R: Training Language Model Agents to Reflect via Iterative Self-Training](http://arxiv.org/abs/2501.11425v1), 2025-01-20

(76) [Neural Contextual Reinforcement Framework for Logical Structure Language Generation](http://arxiv.org/abs/2501.11417v1), 2025-01-20

(77) [Beyond the Hype: Benchmarking LLM-Evolved Heuristics for Bin Packing](http://arxiv.org/abs/2501.11411v1), 2025-01-20

(78) [Revisiting Language Models in Neural News Recommender Systems](http://arxiv.org/abs/2501.11391v1), 2025-01-20

(79) [Towards Advancing Code Generation with Large Language Models: A Research Roadmap](http://arxiv.org/abs/2501.11354v1), 2025-01-20

(80) [EndoChat: Grounded Multimodal Large Language Model for Endoscopic Surgery](http://arxiv.org/abs/2501.11347v1), 2025-01-20

(81) [Few-shot Policy (de)composition in Conversational Question Answering](http://arxiv.org/abs/2501.11335v1), 2025-01-20

(82) [Question-to-Question Retrieval for Hallucination-Free Knowledge Access: An Approach for Wikipedia and Wikidata Question Answering](http://arxiv.org/abs/2501.11301v1), 2025-01-20

(83) [RedStar: Does Scaling Long-CoT Data Unlock Better Slow-Reasoning Systems?](http://arxiv.org/abs/2501.11284v1), 2025-01-20

(84) [Large Language Model Agents for Radio Map Generation and Wireless Network Planning](http://arxiv.org/abs/2501.11283v1), 2025-01-20

(85) [Multi-round, Chain-of-thought Post-editing for Unfaithful Summaries](http://arxiv.org/abs/2501.11273v1), 2025-01-20

(86) [Can xLLMs Understand the Structure of Dialog? Exploring Multilingual Response Generation in Complex Scenarios](http://arxiv.org/abs/2501.11269v1), 2025-01-20

(87) [Code Readability in the Age of Large Language Models: An Industrial Case Study from Atlassian](http://arxiv.org/abs/2501.11264v1), 2025-01-20

(88) [Multivariate Wireless Link Quality Prediction Based on Pre-trained Large Language Models](http://arxiv.org/abs/2501.11247v1), 2025-01-20

(89) [Irony in Emojis: A Comparative Study of Human and LLM Interpretation](http://arxiv.org/abs/2501.11241v1), 2025-01-20

(90) [PlotEdit: Natural Language-Driven Accessible Chart Editing in PDFs via Multimodal LLM Agents](http://arxiv.org/abs/2501.11233v1), 2025-01-20

(91) [KPL: Training-Free Medical Knowledge Mining of Vision-Language Models](http://arxiv.org/abs/2501.11231v1), 2025-01-20

(92) [Reasoning Language Models: A Blueprint](http://arxiv.org/abs/2501.11223v1), 2025-01-20

(93) [Embedding-Driven Diversity Sampling to Improve Few-Shot Synthetic Data Generation](http://arxiv.org/abs/2501.11199v1), 2025-01-20
